src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -2169,6 +2169,11 @@ namespace cryptonote
     return true;
   }
   //-----------------------------------------------------------------------------------------------
+  bool core::set_peer_reachable(crypto::public_key const &pubkey, bool value)
```

Comment:
The method name in core here should be qualified since the concept of a peer in core has a different meaning.  Perhaps `set_storage_server_peer_reachable`?  Or perhaps `set_ss_peer_reachable`?

Reply:
Updated